pattern_manager/PatternParent parent
uint64 num_points
float64 radius
bool tan_rot
bool cw
float64 angular_section
---
bool success